### PR TITLE
Implement stdio debugger connection type

### PR DESCRIPTION
**Pull request by Anthony-Eid**

### CODE REVIEW

File: crates/debugger_ui/src/debugger_panel.rs [hunk 1]
```diff
@@ -272,6 +272,24 @@ impl DebugPanel {
...
+        let all_threads = event.all_threads_continued.unwrap_or(false);
+        let client = this.debug_client_by_id(client_id.clone(), cx);
+
+        if all_threads {
+            for thread in client.thread_states().values_mut() {
+                thread.status = ThreadStatus::Running;
+            }
+        } else {
+            client.update_thread_state_status(event.thread_id, ThreadStatus::Running);
+        }
```
Comment (RemcoSmitsDev) [inline crates/debugger_ui/src/debugger_panel.rs:290]: I think we have to notify here at the end, to make sure all the tabs are re-rendered. And you are not allowed to use any off the buttons for example the `continue` button.

File: crates/debugger_ui/src/debugger_panel.rs [hunk 2]
```diff
@@ -272,6 +272,24 @@ impl DebugPanel {
...
     }
 
+    fn handle_continued_event(
+        this: &mut Self,
+        client_id: &DebugAdapterClientId,
+        event: &ContinuedEvent,
+        cx: &mut ViewContext<Self>,
+    ) {
+        let all_threads = event.all_threads_continued.unwrap_or(false);
+        let client = this.debug_client_by_id(client_id.clone(), cx);
```
Comment (RemcoSmitsDev) [inline crates/debugger_ui/src/debugger_panel.rs]: I think we can deref here, by adding `* client_id` instead of cloning it.

Comment (Anthony-Eid) [reply crates/debugger_ui/src/debugger_panel.rs]: Fixed! Thanks for the feedback

File: crates/debugger_ui/src/debugger_panel.rs (hunk 1 above)
Comment (Anthony-Eid) [reply crates/debugger_ui/src/debugger_panel.rs:290]: Done, thanks!

**Pull request merged**